afc-submit-wizard.js

```diff
@@ -747,7 +922,26 @@ function prepareTalkText(initialText) {
 
 	// remove |class=draft parameter in any WikiProject templates
 	text = text.replace(/(\{\{wikiproject.*?)\|\s*class\s*=\s*draft\s*/gi, '$1');
-
+	
+	sng = ui.sngInput.getValue().split(",");
+	sourcesExist = (sng[0] == "GNG") && (ui.bestsourcesInput1.getValue() || ui.bestsourcesInput2.getValue() || ui.bestsourcesInput3.getValue());
+	
+	if (sourcesExist) {
+		text = text.replace(/\{\{Best sources\|(.*?)\}\}/g, '');
+		bestSources = '\n== [[WP:THREE]] best sources ==\n{{Best sources';
+		if (ui.bestsourcesInput1.getValue()) {
+			bestSources += ' | src1 = ' + ui.bestsourcesInput1.getValue();
+		}
+		if (ui.bestsourcesInput2.getValue()) {
+			bestSources += ' | src2 = ' + ui.bestsourcesInput2.getValue();
+		}
+		if (ui.bestsourcesInput3.getValue()) {
+			bestSources += ' | src3 = ' + ui.bestsourcesInput3.getValue();
+		}
+		bestSources += '}}';
+		text += bestSources;
+	}
```

Comment:
Without a signature, it won't be touched by archiving bots, and cannot be replied to via the reply button.